lib/simulator-xcode-6.js

```diff
@@ -630,8 +630,9 @@ class SimulatorXcode6 extends EventEmitter {
    * @param {object} updates - The hash of key/value pairs to update for Safari.
    */
   async updateSafariSettings (updates) {
-    await settings.updateSafariUserSettings(this, updates);
-    await settings.updateSettings(this, 'mobileSafari', updates);
+    let updated = await settings.updateSafariUserSettings(this, updates);
+    updated = await settings.updateSettings(this, 'mobileSafari', updates) || updated;
+    return updated;
```

Comment:
perhaps `return await settings.updateSafariUserSettings(this, updates) | await settings.updateSettings(this, 'mobileSafari', updates)` ?